fpdf/text_region.py

```diff
@@ -116,12 +153,29 @@ def ln(self, h=None):
         self._text_fragments.append(fragment)
 
     def build_lines(self, print_sh) -> List[LineWrapper]:
+        if self.bullet:
+            bullet_line_break = MultiLineBreak(
+                [self.bullet.fragment],
+                max_width=self._region.get_width,
+                margins=(self.pdf.c_margin, self.pdf.c_margin),
+                align=self.text_align or self._region.text_align or Align.L,
+                indent=self.indent
+                - self.bullet.fragment.get_width()
+                - self.bullet.rel_x_displacement,
+                print_sh=print_sh,
+                wrapmode=self.wrapmode,
+                line_height=self.line_height,
+                skip_leading_spaces=self.skip_leading_spaces
+                or self._region.skip_leading_spaces,
+            )
+            self.bullet.text_line = bullet_line_break.get_line()
```

Comment:
The relevant block of code will be moved to `Paragraph.generate_bullet_frag()` in the next commit.